satellite/gracefulexit/pending.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit
+
+import (
+	"context"
+	"sync"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+)
+
+// PendingFinishedPromise for waiting for information about finished state.
+type PendingFinishedPromise struct {
+	addedWorkChan      chan struct{}
+	finishedCalledChan chan struct{}
```

Comment:
nit - should this be `finishCalledChan` and similarly below with the `finishCalled` method?